extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/extractors/MediaCCCStreamExtractor.java

```diff
@@ -108,11 +111,18 @@ public List<AudioStream> getAudioStreams() throws ExtractionException {
                 } else if (mimeType.endsWith("ogg")) {
                     mediaFormat = MediaFormat.OGG;
                 } else {
-                    throw new ExtractionException("Unknown media format: " + mimeType);
+                    mediaFormat = null;
                 }
 
-                audioStreams.add(new AudioStream(recording.getString("recording_url"),
-                        mediaFormat, -1));
+                // Don't use the containsSimilarStream method because it will always return
+                // false. So if there are multiple audio streams available, only the first one will
+                // be extracted in this case.
+                audioStreams.add(new AudioStream.Builder()
```

Comment:
Uhm, since `add` is called for `recordings.size()` times (i.e. each iteration of the for loop), it is false that "only the first one will be extracted", unless I understand something incorrectly, in which case please rephrase. Also in `getVideoStreams`.

Reply:
@TiA4f8R I can make changes myself about this, but if you have a moment please provide an answer so that I can proceed

Reply:
Ok, this comment is not clear. We don't know the average bitrate of MediaCCC audio streams, nor the language of (video, at least?) streams.

`only the first one will be extracted in this case` mean that if we used `containsSimilarStream` method of `Stream` classes for MediaCCC streams, they would return `true` where they shouldn't, because of some missing information from MediaCCC API, at least in a proper way for a part of the information.

Btw, we should probably ask an API update on their GitHub repository to add the audio average bitrate if possible, content languages (returned in the `tech` string if I am not wrong), and everything else we need that we currently don't extract.

Reply:
Thank you, I improved the comment